src/pixel/state.ts

```diff
@@ -26,6 +26,9 @@ const paramExtractors: ((state: State) => [string, string][])[] = [
   ifDefined('eventSource', source => asParamOrEmpty('se', source, (s) => base64UrlEncode(JSON.stringify(s, replacer)))),
   ifDefined('liveConnectId', fpc => asStringParam('duid', fpc)),
   ifDefined('trackerVersion', v => asStringParam('tv', v)),
+  // For backward compatibility
+  // @ts-ignore
+  ifDefined('trackerName', v => asStringParam('tv', v)),
```

Comment:
In what cases is this required?

Reply:
thanks for asking
I misinterpreted a scenario related to prebid old versions, but we won't run into that case as the old dependency would be imported..